addons/mrp/report/mrp_report_mo_overview.py

```diff
@@ -240,10 +240,12 @@ def get(replenishment, key, check_in_receipt=False):
         reserved_quantity = self._get_reserved_qty(move, warehouse, replenish_data)
         missing_quantity = move.product_uom_qty - reserved_quantity
         free_qty = product.uom_id._compute_quantity(product.free_qty, move.product_uom)
-        if not has_to_order_line and float_compare(missing_quantity, free_qty, precision_rounding=move.product_uom.rounding) <= 0:
+        if not has_to_order_line \
+           and float_compare(missing_quantity, 0.1, 2) <= 0 \
+           or float_compare(missing_quantity, free_qty, precision_rounding=move.product_uom.rounding) <= 0:
```

Comment:
`if missing_qty <= 0 or (not order_lines and missing_qty <= free_qty)`
Note that *in this specific case* parenthesis aren't required (since it matches the order of operations), but I think it's always better to be explicit about the order of the boolean comparisons.

Reply:
Done